### DISCUSSION TITLE
23.12.21 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#95] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 코드리뷰 코멘트 작성
    - 발표 준비
- 내일은 어떤 일을 하나요?
    - 발표 준비

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 리팩토링을 위한 코드 분석
  - 성능 측정 방법에 대한 공부
- 내일은 어떤 일을 하나요?
  - 발표 준비
